### PR TITLE
fix: CE-1050 Makes it so cypress tests work on empty case management database

### PR DESCRIPTION
**Pull request by afwilcox**
# Description

Please provide a summary of the change and the issue fixed. Please include relevant context. List dependency changes.

CE-1106

Contains some updates to Cypress Tests.   In some cases, added some extra identifiers onto the forms in order to make the elements selectable.   

- Update Status from Details view now fills in the assessment section if it is empty preventing a validation error from failing the test.
- Assessment and Equipment tests no longer refer to fictional test users
- An assertion in the File Review Test was updated to reflect the new display name of our test account
- Added a few functions to lower duplication

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [X] Run test suite against empty case management database to ensure all tests pass
- [X] Run tests again against same database to ensure all tests are re-runnable

## Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes

## Further comments

---

Thanks for the PR!

Deployments, as required, will be available below:
- [Frontend](https://nr-compliance-enforcement-695-frontend.apps.silver.devops.gov.bc.ca)
- [Backend](https://nr-compliance-enforcement-695-frontend.apps.silver.devops.gov.bc.ca/api)

Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/nr-compliance-enforcement/actions/workflows/analysis.yml)

After merge, new images are deployed in:
- [Merge Workflow](https://github.com/bcgov/nr-compliance-enforcement/actions/workflows/merge.yml)

---

Thanks for the PR!

Deployments, as required, will be available below:
- [Frontend](https://nr-compliance-enforcement-695-frontend.apps.silver.devops.gov.bc.ca)
- [Backend](https://nr-compliance-enforcement-695-frontend.apps.silver.devops.gov.bc.ca/api)

Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/nr-compliance-enforcement/actions/workflows/analysis.yml)

After merge, new images are deployed in:
- [Merge Workflow](https://github.com/bcgov/nr-compliance-enforcement/actions/workflows/merge.yml)

### CODE REVIEW

File: frontend/cypress/e2e/hwcr-outcome-equipment.cy.ts [hunk 1]
```diff
@@ -8,50 +8,40 @@ describe("HWCR Outcome Equipment", () => {
...
-      if ($equipment.find("#outcome-report-add-equipment").length > 0) {
-        deleteAllEquipments();
-        cy.get("#outcome-report-add-equipment").click();
-      } else {
-        cy.log("Test was previously run. Skip the Test");
-        this.skip();
-      }
-    });
-    cy.get(".comp-outcome-report-complaint-assessment").then(function ($equipment) {
+    //Run before exery test in order to make re-runnable.
```
Comment (nayr974) [inline frontend/cypress/e2e/hwcr-outcome-equipment.cy.ts:26]: <3

File: frontend/cypress/support/commands.ts [hunk 2]
```diff
@@ -127,6 +127,18 @@ Cypress.Commands.add("kcLogout", () => {
   });
 });
 
+Cypress.Commands.add("hasErrorMessage", (inputs: Array<string>, toastText: string) => {
```
Comment (nayr974) [inline frontend/cypress/support/commands.ts:130]: This is slick, but might it obfuscate which input is raising the error versus the explicit checks? It's unclear to me if the input would get logged or if a screenshot would show that

**Review by nayr974 — COMMENTED** [review]
Looks good, proof's in the pudding if the test are passing but I have one question about the changes.

File: frontend/cypress/support/commands.ts (hunk 2 above)
Comment (afwilcox) [reply frontend/cypress/support/commands.ts:130]: Great thought process, I had to try it out with an introduced missing validation message and see what it spat out - thankfully it does log the field it was testing (id=action-required-div in this example) before the error is thrown:

image

Comment (nayr974) [reply frontend/cypress/support/commands.ts:130]: Awesome! Much cleaner then

**Review by nayr974 — APPROVED** [review]

**Pull request merged**